giza/agents/integration.py

```diff
@@ -0,0 +1,13 @@
+from ape.api import AccountAPI
+
+import giza.agents.integrations.uniswap.uniswap as uniswap_module
+
+
+class IntegrationFactory:
+    @staticmethod
+    def from_name(name: str, sender: AccountAPI) -> uniswap_module.Uniswap:
+        match name:
+            case "UniswapV3":
```

Comment:
I think that it would be better to have an enum for this, maybe open an issue and will deal with this later